__tests__/app.test.js

```diff
@@ -36,4 +44,28 @@ describe('# GET REQUESTS', () => {
                 });
         });
     });
+
+    describe('GET /api/atricles/:article_id', () => {
+
+        test('200: Returns list of articles', () => {
+            return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then(res => {
+                    const articles = res.body.articles;
+                    articles.forEach(article => {
+                        
+                        // check object properties
+                        expect(article).toMatchObject({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
```

Comment:
As this request is specifically for `article_id` 1 then you can be more specific here it must respond with the right id, not just any Number.